cloud/Makefile

```diff
@@ -0,0 +1,36 @@
+-include default.env
+
+vpc_id ?= $(eval vpc_id := $(shell bash -c 'read -p "VPC ID (existing): " input; echo $$input'))$(vpc_id)
```

Comment:
Mostly out of curiosity: Why pass things through `bash -c` everywhere?

Reply:
just struggling with how Make handles variables 😄 Otherwise the command output is not properly written through. But I am definitively no Make expert, so if you have a more elegant way to write it, I am all for it 😉